dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -95,10 +95,6 @@ public static void main(String[] arguments)
       throws IOException, TemplateException, RepositoryException, URISyntaxException,
           PlexusContainerException, ComponentLookupException, ProjectBuildingException,
           ParseException {
-    if (arguments.length != 1) {
```

Comment:
This validation was not a problem when Maven-exec-plugin passes arguments. (See https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/578)